src/Sylius/Bundle/AdminBundle/Resources/assets/controllers/DeleteTaxonController.js

```diff
@@ -18,7 +18,7 @@ export default class extends Controller {
             this.csrfTokenTarget.value = event.detail.csrfToken;
             this.modalElement = this.modalTarget;
 
-            this.modalElement.closest('.dropdown').appendChild(this.modalElement);
+            this.modalElement.closest('.btn-group').appendChild(this.modalElement);
```

Comment:
Maybe make a custom attribute for this hook, so we don't need to care if the class changes again?
Could be done similarly w/ the `hidden.bs.modal` below